src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -1063,12 +1063,12 @@ public function __construct(_DictionaryIncrementResponse $response)
         $this->value = $response->getValue();
     }
 
-    public function value(): int
+    public function valueInt(): int
     {
         return $this->value;
     }
 
-    public function string(): string
+    public function valueString(): string
```

Comment:
Would you mind just removing this function please? I added it originally, apparently by mistake.